arangod/IResearch/IResearchViewCoordinator.cpp

```diff
@@ -138,212 +127,162 @@ struct IResearchViewCoordinator::ViewFactory : public arangodb::ViewFactory {
              "arangosearch view '"
           << impl->name() << "'";
     }
-
-    view = ci.getView(
-        vocbase.name(),
-        std::to_string(impl->id().id()));  // refresh view from Agency
-
+    // refresh view from Agency
+    view = ci.getView(vocbase.name(), std::to_string(impl->id().id()));
     if (view) {
-      view->open();  // open view to match the behavior in
-                     // StorageEngine::openExistingDatabase(...) and original
-                     // behavior of TRI_vocbase_t::createView(...)
+      // open view to match the behavior in StorageEngine::openExistingDatabase
+      // and original behavior of TRI_vocbase_t::createView
+      view->open();
     }
-
-    return Result();
+    return {};
   }
 
-  virtual Result instantiate(LogicalView::ptr& view, TRI_vocbase_t& vocbase,
-                             VPackSlice definition) const override {
+  Result instantiate(LogicalView::ptr& view, TRI_vocbase_t& vocbase,
+                     velocypack::Slice definition) const override {
```

Comment:
Why not VPackSlice ?

Reply:
Same as others

Reply:
Maybe better to rename all to VPackSlice

Reply:
VPackSlice has a problem, it's using, so we can't forward declaration it

Reply:
But we use VPackBuilder at line 159. It is also using and even defined in the same file as VPackSlice